tools/clang-tidy-plugin/CMakeLists.txt

```diff
@@ -58,10 +58,6 @@ endif ()
 target_include_directories(${CataAnalyzerName} SYSTEM PRIVATE
         ${LLVM_INCLUDE_DIRS} ${CLANG_INCLUDE_DIRS})
 
-if (NOT "${CATA_CLANG_TIDY_INCLUDE_DIR}" STREQUAL "")
-    target_include_directories(${CataAnalyzerName} SYSTEM PRIVATE ${CATA_CLANG_TIDY_INCLUDE_DIR})
-endif ()
-
```

Comment:
This is required for compiling clang-tidy on Windows.

Reply:
Do you think it makes sense to still have this specific feature rather than just allowing folks to pass an extra include dir via `CMAKE_CXX_FLAGS`?

Reply:
I think `CMAKE_CXX_FLAGS` applies to all targets though. Is there a way to specify include dir for specific targets through the command line?

Reply:
No, I don't think so.  So indeed it's probably sensible to keep this (which I think is what I did end up doing in this PR).